server/util/ServerCommand.java

```diff
@@ -0,0 +1,162 @@
+    class Start implements ServerCommand {
+
+        @Option(descriptionKey = "server.data",
+                names = {"--data"},
+                description = "Directory in which database server data will be stored")
+        private String data;
+
+        @Option(descriptionKey = "server.port",
+                names = {"--port"},
+                defaultValue = ServerDefaults.DEFAULT_DATABASE_PORT + "",
+                description = "Port number of database server in which GRPC clients will connect to")
+        private int port;
+
+        @Option(descriptionKey = "grabl.trace",
+                names = {"--grabl-trace"},
+                negatable = true,
+                defaultValue = "false",
+                description = "Enable Grabl performance tracing")
+        private boolean grablTrace;
+
+        @Option(descriptionKey = "grabl.uri",
+                names = {"--grabl-uri"},
+                description = "Grabl tracing server URI")
+        private URI grablURI;
+
+        @Option(descriptionKey = "grabl.username",
+                names = {"--grabl-username"},
+                description = "Grabl username")
+        private String grablUsername;
+
+        @Option(descriptionKey = "grabl.token",
+                names = {"--grabl-token"},
+                description = "Grabl account access token")
+        private String grablToken;
+
+        @Option(descriptionKey = "debug",
+                names = {"--debug"},
+                description = "Debug mode")
+        private boolean debug;
+
+        public Path dataDir() {
+            if (data == null) return ServerDefaults.DATA_DIR;
+            return Paths.get(data).isAbsolute()
+                    ? Paths.get(data)
+                    : ServerDefaults.GRAKN_DIR.resolve(data);
+        }
+
+        public int port() {
+            return port;
+        }
+
+        public boolean debug() {
+            return debug;
+        }
+
+        public boolean grablTrace() {
+            return grablTrace;
+        }
+
+        public URI grablURI() {
+            return grablURI;
+        }
+
+        public String grablUsername() {
+            return grablUsername;
+        }
+
+        public String grablToken() {
+            return grablToken;
+        }
+
+        @Override
+        public ServerCommand.Start asStart() {
+            return this;
+        }
+    }
+
+    @Command(name = "import")
+    class ImportData implements ServerCommand {
+
+        private final Start startCommand;
+
+        @Parameters(  index = "0", description = "Database to import data into")
+        private String database;
+
+        @Parameters(index = "1", description = "File containing the data to import")
+        private String filename;
+
+        @Parameters(index = "2..*", arity = "0..*", description = "Schema concept remap labels")
+        private Map<String, String> remapLabels = new LinkedHashMap<>();
+
+        public ImportData(Start startCommand) {
```

Comment:
`ImportData` command takes in `Start` command since we may want to do something like:

```
./grakn server --port 2000 import grabl grabl.data
```

We use the options at `Start` command to know where the server is.